### ISSUE TITLE
🐛 Bug Report: GitHub organization selector is not refreshed after setting a new connection

### ISSUE THREAD

**Issue by abrl91** [amplication/amplication#3600]
### What happened?

 GitHub organization selector is not refreshed after setting a new connection.
 

Uploading Screen Recording 2022-08-27 at 21.30.23.mov…

### What you expected to happen

I expect that when I set a new connection, the new connection would be selected automatically

### How to reproduce

- go to connect to GitHub page
- if you don't have a connection, connect one
- then set a connection to another organization
- see that the new connection is not selected automatically

### Amplication version

0.15.0

**Comment by samanteymoori** [comment]
Hi @abrl91 It's nice to meet you, it's my first time as a contributor  I like to start with this issue so I would be happy to help if you could assign me this task

**Comment by abrl91** [comment]
> Hi @abrl91 It's nice to meet you, it's my first time as a contributor I like to start with this issue so I would be happy to help if you could assign me this task

Hey @samanteymoori Nice to meet you too 😃 Good luck!

**Comment by souravjain540** [comment]
@abrl91 is this issue still exists?

**Comment by abrl91** [comment]
> @abrl91 is this issue still exists?

Yes. you add a new organization and the expectation is that this new org will be selected automatically, but instead, you need to select it manually. This happens only when there is more than 1 org

**Comment by 2gacem7** [comment]
Hi @abrl91, @samanteymoori ! Can i work on it ?

**Comment by abrl91** [comment]
@2gacem7 I assigned you. Good luck :)

**Comment by 2gacem7** [comment]
Thank you!

**Comment by MatteoGheza** [comment]
Can I work on this?

**Comment by karthxk07** [comment]
Hey, I will try to work on this, just wanted to let the community know :+1:

**Comment by GreenMachine01** [comment]
@karthxk07 assigned to you. good luck!
